src/data/redux.test.jsx

```diff
@@ -665,13 +668,32 @@ describe('Data layer integration tests', () => {
 
     it('Should call the exams service for update attempt', async () => {
       const updateExamAttemptURL = `${getConfig().EXAMS_BASE_URL}/attempt/${attempt.attempt_id}`;
+      const examURL = `${getConfig().EXAMS_BASE_URL}/exam/attempt/course_id/${courseId}/content_id/${contentId}`;
+      const activeAttemptURL = `${getConfig().EXAMS_BASE_URL}/exams/attempt/latest`;
 
-      axiosMock.onGet(fetchExamAttemptsDataUrl).replyOnce(200, { exam: {}, active_attempt: attempt });
+      axiosMock.onGet(examURL).reply(200, { exam });
+      axiosMock.onGet(activeAttemptURL).reply(200, { attempt });
       axiosMock.onPut(updateExamAttemptURL).reply(200, { exam_attempt_id: attempt.attempt_id });
 
       await executeThunk(thunks.getExamAttemptsData(courseId, contentId), store.dispatch);
       await executeThunk(thunks.stopExam(), store.dispatch, store.getState);
       expect(axiosMock.history.put[0].url).toEqual(updateExamAttemptURL);
     });
+
+    it('Should call the exams service to fetch attempt data', async () => {
+      const examURL = `${getConfig().EXAMS_BASE_URL}/exam/attempt/course_id/${courseId}/content_id/${contentId}`;
+      const activeAttemptURL = `${getConfig().EXAMS_BASE_URL}/exams/attempt/latest`;
+
+      axiosMock.onGet(examURL).reply(200, { exam });
+      axiosMock.onGet(activeAttemptURL).reply(200, { attempt });
+
+      await executeThunk(thunks.getExamAttemptsData(courseId, contentId), store.dispatch);
+
+      expect(axiosMock.history.get[0].url).toEqual(examURL);
+      expect(axiosMock.history.get[1].url).toEqual(activeAttemptURL);
+
+      const state = store.getState();
+      expect(state).toMatchSnapshot();
+    });
```

Comment:
Mostly a clarifying question for myself here, it seems like the edx-proctoring functionality/tests remain the same, and you're just adding in tests for edx-exams?

Reply:
Yep! edx-proctoring behavior is already covered by the existing tests, so I am just adding a test case to cover the behavior for if edx-exams is enabled.